custom_components/unifi_insights/api/network/endpoints/clients.py

```
@@ -311,9 +311,7 @@ async def execute_action(
         }
         command = command_map.get(action)
         if command is None:
-            raise ValueError(
-                f"Action must be one of: {', '.join(sorted(command_map))}"
-            )
+            raise ValueError(f"Action must be one of: {', '.join(sorted(command_map))}")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)